site/Index.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { HashRouter as Router } from 'react-router-dom';
+import { Provider } from 'xfc';
+
+import App from 'terra-site/lib/app/App';
+import routeConfiguration from 'terra-site/lib/app/configureApp';
+
+import componentConfig from './generatedComponentConfig';
+import siteConfig from './siteConfig';
+
+const { routeConfig, navigation } = routeConfiguration(siteConfig, componentConfig);
+
+const routes = Object.freeze(routeConfig);
+
+Provider.init({
+  acls: ['*'],
+  secret: () => (Promise.resolve('Success')),
+});
+
+const themes = {
+  'Default Theme': '',
+  'Mock Theme': 'cerner-mock-theme',
+};
+
+ReactDOM.render((
+  <Router>
+    <App routeConfig={routes} navigation={navigation} rootPath={siteConfig.rootPath} themes={themes} appTitle="Title" appSubtitle="Framework" appLogoSrc={siteConfig.appLogoSrc} />
```

Comment:
The terra-site [`App`](https://github.com/cerner/terra-site/blob/dynamic-site/src/app/App.jsx) component, requires `routeConfig`, `navigation`, and `rootPath`. All remaining props are optional, contain the default values most expected by consumers.